### ISSUE TITLE
'Use System Tray Icon' disables security - 3.5.1

### ISSUE THREAD

**Issue by royjohngless** [winauth/winauth#407]
Hi, I'm not sure if this is intended behavior, or a known issue (didn't see one posted to github), but as the title states, if you have WinAuth set to use a password to open, using the system tray icon will disable the need for password. 

To recreate;

-Open WinAuth
-Select the gear icon on the bottom right
-Check the box that says "Use System Tray Icon"
-Close WinAuth
-Open WinAuth from either the Start Menu, Taskbar, System Tray Icon, etc. 

It will no longer require you to enter your password before opening despite the option to use password being still checked in settings. Simply disabling the use of system tray icon will enable password protection again.

**Comment by winauth** [comment]
This is working as intended. The password is for when it is first opened and the data is loaded, although after that it can be minimised to the tray, but it doesn't required a password to maximise it again.

There has been another suggestion that it expires that password after a certain amount of time, which would clear any loaded data and require re-entering the password. Likewise, it could be set to do that if it were minimised to the system tray. So I'll change this to an enhancement request.

**Comment by My1** [comment]
I agree with @royjohngless I had a similar Idea, although being manual ( #344 ) and especially considering that it is already done when using Yubikey I see no reason not to do it with password.
